roles/container-engine/containerd/defaults/main.yml

```
@@ -13,7 +13,7 @@ containerd_runc_runtime:
   engine: ""
   root: ""
   options:
-    systemCgroup: "true"
+    systemdCgroup: "true"
```

I would propose something better:

```suggestion
    systemdCgroup: "{{ (kubelet_cgroup_driver == 'systemd') | ternary('true', 'false', omit) }}"
```

As it stands, just fixing the typo breaks `kubelet_cgroup_driver: cgroupfs` scenario.